bin/yaml/cobol.yaml

```diff
@@ -0,0 +1,17 @@
+compilers:
+  cobol:
+    cobol:
+      type: tarballs
+      compression: tar
+      dir: gnucobol
+      url: https://sourceforge.net/projects/gnucobol/files/gnucobol/3.2/gnucobol-3.2-rc2.tar.bz2
+      check_exe: bin/cobc --version
+      after_stage_script:
+        - cd {dir}
```

Comment:
Can we move this to its own builder please?

Unfortunately the computer that runs the installs is not necessarily the same architecture and operating system as those that will run the code itself.

We have a number of docker containers we use to build for the correct architecture and OS. Given how straightforward this looks, I'll make one based off this template and link it back so you can see. It will let us build multiple versions too.

Reply:
I added https://github.com/compiler-explorer/cobol-builder

first build was for 3.2-rc2 - https://github.com/compiler-explorer/infra/actions/runs/4333230828/jobs/7566213229

I'll make a change (using this as a template) to install it

Reply:
If you give me a list of gnu cobols to build I can do those too.

Reply:
> If you give me a list of gnu cobols to build I can do those too.

I'd suggest to go to https://ftp.gnu.org/gnu/gnucobol/?C=M;O=D (or a mirror, if that would be built more often) and go with gnu-cobol-1-1, gnucobol-2.2 and gnucobol-3.1.2

Reply:
Cool - if they all build the same way this will be trivial once I have 3.2-rc2 working.

Reply:
https://alpha.gnu.org/gnu/gnucobol/ does not contain a 1.1, so I'm going with 2.2 and 3.1.2

Reply:
2.2-rc2 failed with:
```
  configure:  COB_EXPORT_DYN    -Wl,--export-dynamic
  configure:  COB_STRIP_CMD     strip --strip-unneeded
  configure:  Dynamic loading:                             System
  configure:  Use gettext for international messages:      yes
  configure:  Use fcntl for file locking:                  yes
  configure:  screen I/O (disabled):                       NO
  configure:  Use Berkeley DB for ISAM I/O:                yes
  ++ nproc
  + make -j48
  CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/bash /root/build/build_aux/missing autoconf
  /root/build/build_aux/missing: line 81: autoconf: command not found
  WARNING: 'autoconf' is missing on your system.
           You should only need it if you modified 'configure.ac',
           or m4 files included by it.
           The 'autoconf' program is part of the GNU Autoconf package:
           <http://www.gnu.org/software/autoconf/>
           It also requires GNU m4 and Perl in order to run:
           <http://www.gnu.org/software/m4/>
           <http://www.perl.org/>
  make: *** [Makefile:488: configure] Error 127
  Error: Process completed with exit code 2.```

Reply:
> https://alpha.gnu.org/gnu/gnucobol/ does not contain a 1.1, so I'm going with 2.2 and 3.1.2

As noted above:

> I'd suggest to go to https://ftp.gnu.org/gnu/gnucobol/?C=M;O=D (or a mirror, if that would be built more often) and go with gnu-cobol-1-1, gnucobol-2.2 and gnucobol-3.1.2

alpha.gnu.org only contains pre-release versions!

> 2.2-rc2 failed with: [...]

That's bad, does the 2.2 final has the same issue? if yes that would be a reason to publish a 2.2.1.